smack-websocket/src/main/java/org/jivesoftware/smack/websocket/LoggingInterceptor.java

```diff
@@ -0,0 +1,42 @@
+/**
+ *
+ * Copyright 2020 Aditya Borikar
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smack.websocket;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public final class LoggingInterceptor implements Interceptor {
+      @Override public Response intercept(Interceptor.Chain chain) throws IOException {
+        Request request = chain.request();
+
+        long t1 = System.nanoTime();
+        Logger.getAnonymousLogger().info(String.format("Sending request %s on %s%n%s",
```

Comment:
The target for this should be `SmackDebugger`, which means that `LoggingInterceptor` probably needs to take an `SmackDebugger` instance as argument.